src/Serializer/src/SerializerManager.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Spiral\Serializer;
+
+class SerializerManager implements SerializerInterface
+{
+    public function __construct(
+        protected readonly SerializerRegistry $serializers,
+        protected readonly string $defaultFormat
+    ) {
+    }
+
+    public function getSerializer(string $format = null): SerializerInterface
+    {
+        return $this->serializers->get($format ?? $this->defaultFormat);
+    }
+
+    public function serialize(mixed $payload, ?string $format = null): string|\Stringable
+    {
+        return $this->getSerializer($format ?? $this->defaultFormat)->serialize($payload, $format);
+    }
+
+    public function unserialize(
+        string|\Stringable $payload,
+        string|object|null $type = null,
+        ?string $format = null
+    ): mixed {
+        return $this->getSerializer($format ?? $this->defaultFormat)->unserialize($payload, $type, $format);
```

Comment:
```suggestion
        return $this->getSerializer($format ?? $this->defaultFormat)->unserialize($payload, $type);
```
Should `$format` be here in the `unserialize` calling?

Reply:
This is a mistake, removed